test/test_StanfordSentimentTreebank.jl

```diff
@@ -0,0 +1,28 @@
+using CorpusLoaders
+using CorpusLoaders: @NestedVector
+using DataDeps
+using Test
+using Base.Iterators
+using MultiResolutionIterators
+
+@testset "Stanford Sentiment Treebank" begin
+    sst_gen = load(StanfordSentimentTreebank())
+    docs = collect(take(sst_gen, 5));
+
+    @test typeof(docs) == @NestedVector(Any, 2)
+    @testset "output test" for doc in docs
+        @test typeof(doc[1]) == @NestedVector(String, 2)
+        @test typeof(doc[2]) <: Number
+    end
+
+    docs = permutedims(reshape(hcat(docs...), (length(docs[1]), length(docs))))
```

Comment:
Isn't this the same as:
```suggestion
    docs = permutedims(hcat(docs...))
```

Reply:
yeah , sillly me